integration/tutorial/iogroup/Makefile

```diff
@@ -12,4 +12,17 @@ libgeopmiogroup_example_iogroup.so.2.0.0: ExampleIOGroup.o
 clean:
 	rm -f ./*.o ./*.d ./*.so*
 
-.PHONY: clean
+check: libgeopmiogroup_example_iogroup.so.2.0.0
+	rm -f /tmp/geopm_example_control.$$(id -u)
+	GEOPM_PLUGIN_PATH=$$PWD geopmread | grep -q TMP_FILE_CONTROL || true # File does not exist, but control is listed
+	echo foobar > /tmp/geopm_example_control.$$(id -u)
+	GEOPM_PLUGIN_PATH=$$PWD geopmread | grep -q TMP_FILE_CONTROL || true # File has bad value, but control is listed
+	echo 42 > /tmp/geopm_example_control.$$(id -u)
+	GEOPM_PLUGIN_PATH=$$PWD geopmread | grep -q TMP_FILE_CONTROL # File has good value, but control is not listed
```

Comment:
Were the the ` || true` parts just to let debugging proceed past failures? Should it check for an expected grep failure instead?